python/core/auto_generated/processing/qgsprocessingparameters.sip.in

```diff
@@ -1525,6 +1559,68 @@ Creates a new parameter using the definition from a script code.
 
 };
 
+class QgsProcessingParameterGeometry : QgsProcessingParameterDefinition
+{
+%Docstring
+A geometry parameter for processing algorithms.
+
+.. versionadded:: 3.16
+%End
+
+%TypeHeaderCode
+#include "qgsprocessingparameters.h"
+%End
+  public:
+
+    QgsProcessingParameterGeometry( const QString &name, const QString &description = QString(), const QVariant &defaultValue = QVariant(), bool optional = false, const QList<QgsWkbTypes::GeometryType> &geometryTypes = QList<QgsWkbTypes::GeometryType>() );
```

Comment:
Can you double check that this is callable with a list of geometry types from python? Elsewhere this has caused crashes, and we've need to use QList< int > instead of QList< actual enum type > for the bindings instead to avoid this crash.

Reply:
You right, that's seem to cause problems with sip - in my case it does not crash but fail to correctly build the argument list.
Changed to QList<int>, added python test.